features/lorawan/lorastack/mac/LoRaMac.cpp

```diff
@@ -1684,12 +1704,14 @@ void LoRaMac::set_tx_continuous_wave(uint8_t channel, int8_t datarate, int8_t tx
     _lora_phy->set_tx_cont_mode(&continuous_wave);
 }
 
-lorawan_status_t LoRaMac::initialize(EventQueue *queue)
+lorawan_status_t LoRaMac::initialize(EventQueue *queue,
+                                     mbed::Callback<void(void)>scheduling_failure_handler)
 {
     _lora_time.activate_timer_subsystem(queue);
     _lora_phy->initialize(&_lora_time);
 
     _ev_queue = queue;
+    _scheduling_failure_handler = scheduling_failure_handler;
```

Comment:
Is it fair to say that a null function pointer will never be passed to this function?

Reply:
It is an internal API and we always pass a legitimate pointer of the callback that's why probably we don't need a null check here.